device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubSendTask.java

```diff
@@ -36,7 +36,8 @@ public IotHubSendTask(IotHubTransport transport)
 
     public void run()
     {
-        Thread.currentThread().setName(THREAD_NAME);
+        String threadName = this.transport.getDeviceClientUniqueIdentifier() + "-" + "Cxn" + transport.getTransportConnectionId() + "-" + THREAD_NAME;
```

Comment:
What does "Cxn" stand for in these thread names?

Reply:
Connection